bot.go

```diff
@@ -61,70 +72,30 @@ func main() { /*
 	discord.AddHandler(serverJoin)
 	discord.AddHandler(serverLeave)
 	discord.AddHandler(onReady)
-	err = botInit()
-	if err != nil {
-		fmt.Println("Error in botinit:", err)
-		return
-	}
```

Comment:
this function was killed and it was turned into a global map literal (commands.go:34)